includes/payment-methods/class-wc-gateway-stripe-bancontact.php

```diff
@@ -125,7 +125,7 @@ public function check_environment() {
 	 */
 	public function get_environment_warning() {
 		if ( 'yes' === $this->enabled && ! in_array( get_woocommerce_currency(), $this->get_supported_currency() ) ) {
-			$message = __( 'Bancontact is enabled - it requires store currency to be set to Euros.', 'woocommerce-gateway-stripe' );
+			$message = sprintf( __( '%1$s is enabled - it requires store currency to be set to %2$s.', 'woocommerce-gateway-stripe' ), __( 'Bancontact', 'woocommerce-gateway-stripe' ), __( 'Euros', 'woocommerce-gateway-stripe' ) );
```

Comment:
Because you are always repeating "some gateway is enabled - it requires store currency to be set to some currency".

I guess it makes sense, but if you don't agree, that's ok :-)

Reply:
Of course, we could also use $this->method_title and some output from get_supported_currency() instead of text

Reply:
Ok so initially I thought you're saying we're doing something "wrong" as in we're not using placeholders when using variables inside a gettext. I would suggest you checkout release/4.1 branch as these notices has been revised. So you may need to close this PR and re-check the branch as its been extracted into its own method.

Reply:
Oh my... I used the master instead of 4.1 O.o
Shame on me. Sorry.

So, this one is already using placeholders, so I'm not going to touch it:
https://github.com/woocommerce/woocommerce-gateway-stripe/blob/release/4.1/includes/admin/class-wc-stripe-admin-notices.php#L171

Regarding these (for each payment method), should I go ahead, taking into account that the placeholders will be replaced by text and not variables?
- https://github.com/woocommerce/woocommerce-gateway-stripe/blob/release/4.1/includes/admin/stripe-alipay-settings.php#L9
- https://github.com/woocommerce/woocommerce-gateway-stripe/blob/release/4.1/includes/admin/stripe-alipay-settings.php#L13
I just think it's more elegant and easy to translate, not that it's done the "wrong way" today.

Reply:
I think perhaps the only thing missing from the way it is done now is `/* translators: description */` text. I don't see a good reason to make changes to those other than if there was a variable used in gettext when it should have been a placeholder.

Reply:
OK, let's close this then.